migrations/sql/V2019.04.18.00.54__add_subscription_table.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE subscription
+(
+    subscription_id serial PRIMARY KEY,
+    idir character varying(30) NOT NULL,
+    mine_guid uuid NOT NULL,
+
+    FOREIGN KEY (mine_guid) REFERENCES mine(mine_guid),
+    CONSTRAINT unq_idir_mine_guid UNIQUE(idir,mine_guid)
+);
+
```

Comment:
File got renamed, copying some of my earlier comments:   
 - I feel the 'idir' column should be more generic and just be 'user_name' or something similar.
 - User name fields in our database are character varying(60), this should follow suit.
 - Please add `ALTER TABLE notification OWNER TO mds;` to line 10

Reply:
done